app/src/main/java/com/framgia/musixmatch/application/MusixMatchApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.framgia.musixmatch.application;
+
+import android.app.Application;
+
+public class MusixMatchApplication extends Application {
+    private static MusixMatchApplication sMusixMatchApplication;
```

Comment:
```suggestion
    private static MusixMatchApplication sApplication;
```
nếu k có nhiều biến dễ bị nhầm lẫn e có thể đặt tên ntnt